### ISSUE TITLE
Render Accordion Title as heading

### ISSUE THREAD

**Issue by tqureshi-uog** [ccswbs/ugnext#261]
Current:

```
      {sectionTitle && (
        <Typography id={`accordions-heading-${data.uuid}`} type={`h${level}` as TypographyProps<any>["type"]}>
          {sectionTitle}
        </Typography>
      )}
```

This renders the accordion title as a `<span>` instead of a heading which results in excessively tight spacing and poor accessibility. It should be refactored so it's consistent with other widget title handling, e.g. section-widget.tsx

```
      {data.heading && (
        <Typography id={`section-heading-${data.uuid}`} type={(data.headingLevel as "h1" | "h2" | "h3" | "h4" | "h5" | "h6") || "h2"} as={(data.headingLevel as "h1" | "h2" | "h3" | "h4" | "h5" | "h6") || "h2"}>
          {data.heading}
        </Typography>
      )}
```

Also, the id should be a slugified version of `data.heading`
